src/app/components/Svgs/BookSvg.jsx

```diff
@@ -3,57 +3,16 @@ import React from 'react';
 
 const BookSvg = () => (
```

Comment:
This should eventually go into https://github.com/NYPL/dgx-svg-icons

Reply:
we are removing nypl libraries, so not sure about that.

Reply:
I understand the move away from the design toolkit but these are svg icons that are agnostic to any design system. Why the move away from this?

Reply:
I agree that we don't need to move icons to a different place for NYPL projects. Any of the SVG images we use should be consolidated in that repository for use by all projects. Are there standards regarding these? US.gov uses FontAwesome which I presume we could add free tier icons in the dgx-svg-icons repo, no?